packages/bundle-size/src/utils/compareResultsInReports.js

```diff
@@ -0,0 +1,46 @@
+const calculateDiffByMetric = require('./calculateDiffByMetric');
+
+/** @typedef {import('./calculateDiffByMetric').DiffByMetric} DiffByMetric */
+/** @typedef {{ empty: boolean, minified: DiffByMetric, gzip: DiffByMetric }} DiffForEntry */
+
+/** @typedef {import("./collectLocalReport").BundleSizeReportEntry & { diff: DiffForEntry }} ComparedReportEntry */
```

Comment:
we are using this on 3 places -> makes sense to create local type via typedef

Reply:
Didn't get this suggestion: it's already a local `typedef`...